bazaar/core/tasks.py

```diff
@@ -414,12 +414,16 @@ def ssdeep_analysis(sha256):
                     try:
                         apk.extract(member, tmp_dir)
                     except Exception as e:
-                        logging.error('Can not extract member: %s due to error %s', member, e)
+                        logging.error('Can not extract member: %s due to an error %s', member, e)
```

Comment:
Hi, thanks for the PR :)

I think this wouldn't work, it should be something along `'%s %s' % (member, e)`. 
However it would be preferable to use python's f-strings such as `f'{member} {e}'`

This is an issue in all the calls to `logging.info()` and `logging.error()`

Reply:
Thanks for review!
According to `it should be something along`:
```
>>> import logging
>>> logging.error("test %s : %s", 1, 2)
ERROR:root:test 1 : 2
```
Related to `python's f-strings ` let me share some part of  [documentation](https://docs.python.org/3/howto/logging.html#optimization).

Long story short -- f-strings are interpolated each every time for the logger, even if logging is not configured for some Logging level.

But If you prefer f-style of course I can change it.
Thanks again!

Reply:
Hmm, ok, I didn't know that. We can leave it as is then.

Reply:
Cool!
I know that this is not real optimization in such case, but I've spent some time with loggers so now always doing that.
Thanks!